package.json

```diff
@@ -21,6 +21,10 @@
 	},
 	"homepage": "https://github.com/OpenLake/Cross-Platform-Activity-Tracker#readme",
 	"devDependencies": {
-		"prettier": "^2.3.1"
+		"prettier": "^2.3.1",
+		"express": "^4.17.1",
+		"mongodb": "^4.0.1",
+		"mongoose": "^5.13.3",
+		"morgan": "^1.10.0"
```

Comment:
These should be in a separate package.json for server. Create a new `server` directory under `packages/` and init a package using `pnpm init`.  Move all server code to this new directory.